docs/FAQ.md

```diff
@@ -288,6 +289,38 @@ For a refresher on Go's recommended workspace organization, see the ["How To Wri
 `dep init -gopath`, which falls back to network mode when the packages are not
 found in `GOPATH`. `dep ensure` doesn't work with projects in `GOPATH`.
 
+## How do I use `dep` with Docker?
+
+`dep ensure -vendor-only` creates the vendor folder from a valid `Gopkg.toml` and `Gopkg.lock` without checking for Go code.
+This is especially useful for builds inside docker utilizing cache layers.
+
+Sample dockerfile:
+
+    FROM golang:1.8 AS builder
+
+    RUN apt-get update && apt-get install -y unzip --no-install-recommends && \
```

Comment:
New releases will be available as bare binary (not in zip) so there is no need to install unzip. sha256 sum will also be available with binary, so no need to hardcode hash below. Most probably downloaded binary will need execute permission `chmod +x`.